fury/actors/effect_manager.py

```diff
@@ -0,0 +1,431 @@
+import os
+import numpy as np
+from fury.actor import Actor, billboard
+from fury.colormap import create_colormap
+from fury.io import load_image
+from fury.lib import Texture, WindowToImageFilter
+from fury.shaders import (attribute_to_actor,
+                          compose_shader,
+                          import_fury_shader,
+                          shader_apply_effects,
+                          shader_custom_uniforms)
+from fury.ui import LineSlider2D, TextBlock2D, Panel2D
+from fury.utils import rgb_to_vtk
+from fury.window import (gl_disable_depth,
+                         gl_set_additive_blending,
+                         RenderWindow,
+                         Scene,
+                         ShowManager)
+
+
+WRAP_MODE_DIC = {"clamptoedge" : Texture.ClampToEdge,
+                 "repeat" : Texture.Repeat,
+                 "mirroredrepeat" : Texture.MirroredRepeat,
+                 "clamptoborder" : Texture.ClampToBorder}
+
+BLENDING_MODE_DIC = {"none" : 0, "replace" : 1,
+                     "modulate" : 2, "add" : 3,
+                     "addsigned" : 4, "interpolate" : 5,
+                     "subtract" : 6}
+
+def window_to_texture(
+        window : RenderWindow,
+        texture_name : str,
+        target_actor : Actor,
+        blending_mode : str = "None",
+        wrap_mode : str = "ClampToBorder",
+        border_color : tuple = (
+            0.0,
+            0.0,
+            0.0,
+            1.0),
+        interpolate : bool = True,
+        d_type : str = "rgb"):
+    """Capture a rendered window and pass it as a texture to the given actor.
+    Parameters
```

Comment:
Change 

```
  """Capture a rendered window and pass it as a texture to the given actor.
    Parameters
```
to

```
  """Capture a rendered window and pass it as a texture to the given actor.
   
    Parameters
```
Docstrings follow a set of well-defined patterns, allowing for automatic documentation generation through tools like Sphinx.